cli/src/main/java/com/devonfw/tools/ide/io/FileAccessImpl.java

```diff
@@ -501,12 +503,12 @@ public void extract(Path archiveFile, Path targetDir, Consumer<Path> postExtract
 
     if (Files.isDirectory(archiveFile)) {
       Path properInstallDir = archiveFile; // getProperInstallationSubDirOf(archiveFile, archiveFile);
-      if (extract) {
-        this.context.warning("Found directory for download at {} hence copying without extraction!", archiveFile);
-        copy(properInstallDir, targetDir, FileCopyMode.COPY_TREE_OVERRIDE_TREE);
-      } else {
-        move(properInstallDir, targetDir);
-      }
+      //      if (extract) {
```

Comment:
Do not commit out-commented-code. I thought this was already in our coding-conventions coming from devonfw but I had to add it here:
https://github.com/devonfw/IDEasy/blob/main/documentation/coding-conventions.adoc#clean-code
```suggestion
```